boards/x220/x220.config

```diff
@@ -28,3 +28,5 @@ export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on"
 export CONFIG_BOOT_KERNEL_REMOVE="quiet"
 export CONFIG_BOOT_DEV="/dev/sda1"
 export CONFIG_USB_BOOT_DEV="/dev/sdb1"
+export FLASHROM_OPTIONS='--force --noverify-all -p internal --ifd --image bios -c MX25L6405D'
```

Comment:
@sebastianmcmillan: `-p internal:laptop=force_I_want_a_brick` doesn't fit the bill?

Reply:
@sebastianMcMillan: not sure about `--noverify-all` either in boards configs, internal logic of flashrom 1.0 verifying and applying another write function if content differs, else hacing a chance of bricking device. Can't test for the kgpe-d16 right now unfortunately, but that is what flashes my mind when reviewing all flashrom commands all at once (behavior changed since flashrom 0.9 with also the possibility of specifying --ifd instead of providing a manual flashmap like it was before)

Reply:
This has been resolved.